cmd/otns-real-proxy/otns-real-proxy.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"github.com/chzyer/readline"
+	"github.com/simonlingoogle/go-simplelogger"
```

Comment:
Do imports need to go in different groups separated by empty lines?

Reply:
[Lint](https://github.com/openthread/ot-ns/pull/37/checks?check_run_id=816176869) complained about it.